packages/commercetools/api-client/package.json

```diff
@@ -5,7 +5,7 @@
   "server": "server/index.js",
   "main": "lib/index.cjs.js",
   "module": "lib/index.es.js",
-  "types": "lib/index.d.ts",
+  "types": "lib/commercetools/api-client/src/index.d.ts",
```

Comment:
This still is incorrect. Once this package is built, the types are available under the `lib/index.d.ts` path

Reply:
I'm not sure 
![Zrzut ekranu 2021-08-16 o 11 29 44](https://user-images.githubusercontent.com/40273258/129542303-69f538f4-6d50-4d31-98d5-475262bb0e4a.png)

Reply:
Updated:

![Zrzut ekranu 2021-08-16 o 13 57 39](https://user-images.githubusercontent.com/40273258/129560144-879ce243-c79b-4a32-9474-f0e9eec3735e.png)